modules/github/__init__.py

```diff
@@ -563,7 +567,7 @@ def check_run(self, data):
             started_at = self._iso8601(data["check_run"]["started_at"])
             completed_at = self._iso8601(data["check_run"]["completed_at"])
             if completed_at > started_at:
-                seconds = (completed_at-started_at).total_seconds()
+                seconds = math.floor((completed_at-started_at).total_seconds())
```

Comment:
No - we want fractions of seconds. Some CI builds can take less than a second.

Reply:
oh alright no worries, it was nagging me for an int instead of a float, no issues

Reply:
oh. that's a good point. `utils.to_pretty_time` should instead accept a float and spit out prettified milliseconds

Reply:
I just reverted it, the next line does exactly that function ( ._.)